### DISCUSSION TITLE
Restore failing with no obvious reason

### DISCUSSION THREAD

**Discussion by OwenPattison** [datalust/seq-tickets#1146] [Help]
Hello, 

I've been having some issues with my VM running seq to the point where I needed to set up a new VM and transfer the data disk, the problem I have encountered is when trying to restore the backup I get the following error:

Seq Version: 5.1.3118

![image](https://user-images.githubusercontent.com/5864244/109293701-10be6100-7824-11eb-93c3-31c9089f17bf.png)
![image](https://user-images.githubusercontent.com/5864244/109292969-110a2c80-7823-11eb-919a-31ca98b59278.png)

any suggestions on this would be greatly appreciated thank you!

**Comment by nblumhardt** [comment]
Hi Owen! This error usually means that the key specified doesn't match the one used to encrypt the backup, unfortunately.

Seq 5.x had and issue whereby, if you have Seq installed in a non-default location (i.e. not `C:\ProgramData\Seq`), then you also needed to specify this path for `seq show-key`, e.g.:

```
seq show-key --storage="F:\Seq"
```

(If Seq was installed to `F:\Seq`, for example.)

Does this help?

Best regards,
Nick

**Reply by nblumhardt** [reply]
Looks like this might be `F:\ProgramData\Seq` in your example.

**Reply by OwenPattison** [reply]
Hey Nick,

thanks for getting back to me, that's useful to know thank you - in which case I wonder if you have the answer to this:

If I can still access the drive seq was installed on, would there be a chance I can recover the key? or is the encryption tied to the VM in some way, the reason being the VM has completely stopped allowing RDP connections - but I can move the disk that was attached to it, onto a different VM.

Thanks again that's helped clarify part of the problem.

Owen

**Reply by nblumhardt** [reply]
Hi Owen,

Unfortunately no, not that I'm aware of - unless it's also the operating system install drive.

A better bet might be to simply attach the disk to the new VM, or copy everything across (you can leave the Stream folder behind, if it's too big), and then use:

```
seq service stop
seq service uninstall
seq service install --storage="..."
seq service start
```

where the argument to `--storage` is the new location you copied Seq folder to.

It's likely this will result in a few errors in things like apps that rely on the encryption key for saved passwords (and authentication, if you're using AAD or OIDC), but if all goes well, Seq will start up and we'll be able to move things forward from there.

Let me know if this helps,
Nick
